CHANGELOG.md

```diff
@@ -1,3 +1,193 @@
+## v3
+
+### v3.0.0 - 2023-11-20
+
+([full changelog](https://github.com/jupyterhub/action-major-minor-tag-calculator/compare/v2.0.0...v3.0.0))
+
+#### Breaking changes
+
+This action now uses Node.js 20.
+
+## Maintenance and upkeep improvements
```

Comment:
`github-activity --heading-level=3` makes the output heading levels be correct. This and some other heading levels below needs `##` added.

Reply:
Done, and I've removed the dependabot updates and linked out to them instead https://github.com/jupyterhub/action-major-minor-tag-calculator/pulls?q=is%3Apr+merged%3A2021-10-25..2023-11-20+author%3Aapp%2Fdependabot